Assembler.Base/DefaultAssemblerFactory.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections.Generic;
+using Assembler.Base.Creators;
+using Assembler.Base.FrameHandlers;
+using Assembler.Base.MessageEnrichers;
+using Assembler.Base.Releasing;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Assembler.Core.RawAssemblingEntities;
+using Microsoft.Extensions.Logging;
+
+namespace Assembler.Base
+{
+    public class DefaultAssemblerFactory<TFrame, TMessageInAssembly>
```

Comment:
Thats really sounds like static class with static functions, sounds much more neat and correct

beside that, really good class, I liked the idea 🐷

Reply:
It's gonna be a builder, I'm gonna rebuild the class

Reply:
OK, what does `builder` means? the fluent-syntax: `.WithENricher().WIthValidtor.WIth...` this?

Reply:
Yes